xtable-service/src/main/java/org/apache/xtable/service/models/ConvertTableResponse.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+package org.apache.xtable.service.models;
+
+import java.util.List;
+
+import lombok.Builder;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@Builder
+public class ConvertTableResponse {
+  @JsonProperty("conversions")
+  private List<RestTargetTable> conversions;
```

Comment:
`convertedTables` maybe a better name?

Reply:
In the spec review it seemed we are ok with this naming https://github.com/apache/incubator-xtable/pull/705

Im not so opposed to making these changes if you feel strongly, its just that we will need to make sure to amend spec.

Reply:
will amend spec in a followup pr